### PR TITLE
Validating bank accounts using exceptions

### PR DESCRIPTION
**Pull request by Stingrayoliv**
https://github.com/tel-ran-de/BE12-Basics/issues/10

### CODE REVIEW

File: src/homeworkAll/TaskExceptions/Main.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        // Клиент должен быть не моложе 17 лет
+        // Банковский iban должен быть длиной 16
+        // Первые два символа iban должны быть "DE"
+        //Программа должна выводить все ошибки. То есть, например, если у пользователя есть банковский счет длины 10 и
+        //начинающийся на "IT", то должно печататься два сообщения об ошибке.
+        List<Customer> customers = Arrays.asList(
+                new Customer("Olga", 14, new BankAccount("IT77777788889")),// 3 ошибки
+                new Customer("Oleg", 88, new BankAccount("IE777777")),// 2 ошибки
+                new Customer("Lora", 17, new BankAccount("DE77777788889999")));// без ошибок
+        for (Customer customer : customers) {
```
Comment (sergeylukichev) [inline src/homeworkAll/TaskExceptions/Main.java:19]: This should be a separate method. The main method should call this method.

**Review by sergeylukichev — APPROVED** [review]
👍

File: src/moduleSecond/lesson29102020ReadingSCVFile/DateTasks.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package moduleSecond.lesson29102020;
+package moduleSecond.lesson29102020ReadingSCVFile;
 
 import java.time.LocalDate;
```
Comment (sergeylukichev) [inline src/moduleSecond/lesson29102020ReadingSCVFile/DateTasks.java:3]: why this is here?

File: src/moduleSecond/lesson29102020ReadingSCVFile/Demo.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import java.nio.file.Paths;
+
+public class Demo {
+    public static void main(String[] args) throws IOException {
+        //variant 1
+        // самый популярный метод в Java 8
+        //Files.lines(Paths.get("test.txt")).forEach(System.out::println);
+
+        // variant 2 если BufferedReader добавляем в try(), то не нужно делать close
+        try(BufferedReader br = new BufferedReader(new FileReader("test.txt"))) {
```
Comment (sergeylukichev) [inline src/moduleSecond/lesson29102020ReadingSCVFile/Demo.java:16]: this is not relevant to the exception task

**Review by sergeylukichev — APPROVED** [review]
👍

File: src/moduleSecond/lesson29102020ReadingSCVFile/DateTasks.java (hunk 2 above)
Comment (Stingrayoliv) [reply src/moduleSecond/lesson29102020ReadingSCVFile/DateTasks.java:3]: Classwork was added by mistake to this branch. Homework has already been checked. Thanks for the comments.

**Pull request closed without merging**